californium-core/src/main/java/org/eclipse/californium/core/network/Exchange.java

```diff
@@ -466,8 +489,9 @@ public boolean isComplete() {
 	public void setComplete() {
 		this.complete = true;
 		ExchangeObserver obs = this.observer;
-		if (obs != null)
+		if (null != obs) {
```

Comment:
I don't have a problem with that at all. I do not understand, though, how this is related. Is it?